src/hydration/thirst.ts

```diff
@@ -1,18 +1,97 @@
 import { Bukkit, GameMode } from 'org.bukkit';
 import { Player } from 'org.bukkit.entity';
 import { PotionEffect, PotionEffectType } from 'org.bukkit.potion';
+import { addHydration, getHydration } from './expbar';
 
-// How often players' hydration level is decreased (seconds)
-const THIRST_CYCLE_DURATION = 60;
+/**
+ * How often players' hydration level is decreased (seconds)
+ * If duration is 60s and hydration -0.01, the bar will last 100 mins
+ */
+const THIRST_CYCLE_DURATION = 3;
```

Comment:
This probably should be 30? 5 minutes is a pretty short time period.

Reply:
Oh, this should have been unchanged. I will change it back to 60

Reply:
Oh, this should have been unchanged, so I will return it back to 60